src/index.ts

```diff
@@ -28,6 +28,7 @@ export { Image, ImageElement } from 'Image/Image';
 export { InfoBox } from 'InfoBox/InfoBox';
 export { default as Label } from 'Label';
 export { default as Link } from 'Link';
+export { ExternalLinkMarker } from 'Link';
```

Comment:
Is it a correct way, or should we do 
`export {Link, ExternalLinkMarker} from 'Link/Link'`

Reply:
You can combine lines 30 and 31 as follows:

```
import { Link, ExternalLinkMarker } from '@wellcometrust/corporate-components';
```

Reply:
Updated :)